include/linux/soundwire/sdw.h

```diff
@@ -931,6 +932,7 @@ struct sdw_bus {
 	u32 bank_switch_timeout;
 	bool multi_link;
 	int hw_sync_min_links;
+	int stream_refcount;
```

Comment:
refcount is native supported by Linux kernel. How about to use it ? It can help multi-thread risk
````
refcount_inc, refcount_set, refcount_dec_and_test.
````